graphql-schemas/schema.yml

```diff
@@ -2441,6 +2441,24 @@ confs:
   fields:
   - { name: slack, type: boolean }
 
+- name: AWSShardSpecOverride_v1
+  interface: IntegrationSpecShardSpecOverride_v1
+  fields:
+    - { name: accountType, type: string, isRequired: true }
+    - { name: awsAccount, type: AWSAccount_v1, isRequired: true }
+    - { name: imageRef, type: string, isRequired: true }
+
+- name: IntegrationSpecShardSpecOverride_v1
+  isInterface: true
+  interfaceResolve:
+    strategy: fieldMap
+    field: accountType
```

Comment:
another sharding strategies we will have in the future is going to be `cluster` so i wonder if this discriminator field could be named `shardingStrategy` instead to cover the more generic intend.

Reply:
Rename the fields but keep the interface as is, right?

Reply:
yes

Reply:
okay, done